config.json

```diff
@@ -2430,6 +2430,21 @@
         ],
         "difficulty": 4
       },
+      {
+        "slug": "complex-numbers",
+        "name": "Complex Numbers",
+        "uuid": "0cc86a92-7118-4cb9-8417-db4557baf364",
+        "prerequisites": [
+          "pattern-matching",
+          "erlang-libraries",
+          "floating-point-numbers"
+        ],
+        "practices": [
+          "pattern-matching",
```

Comment:
Should I remove pattern matching? It's not crucial at all. Maybe replace by `erlang-libraries` because of the `:math` functions?